external/SPHEREPACK/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+enable_language(Fortran)
+set(LIBRARY SPHEREPACK)
+
+set(LIBRARY_SOURCES
+    alf.f
+    divgs.f
+    gaqd.f
+    gradgs.f
+    hrfft.f
+    shags.f
+    shsgs.f
+    slapgs.f
+    sphcom.f
+    vhags.f
+    vhsgs.f
+    vrtgs.f
+    vtsgs.f
+    )
+
+add_library(${LIBRARY} ${LIBRARY_SOURCES})
+
```

Comment:
remove these last three lines (we are building the library dependencies differently)